source/application-platform.html.md.erb

```diff
@@ -0,0 +1,153 @@
+
+#### Adopt common standards
+
+Innovation is encouraged but we don't want teams diverging too much. Otherwise we end up with many solutions to the same problems causing a maintenance headache for both developers and operations.
+
+By using standard components available in the Application Platform we can manage services in a consistent way and share best practice.
+
+#### Increase institutional knowledge
+
+As the trickle of new services turns into a flood we find that many projects are at different levels of maturity. These then require different levels of support from teams of different sizes.
+
+By using standard components we share knowledge of our development practices. This makes it easier to move developers between teams and provide the right levels of support where needed.
+
+#### Secure services
+
+Security is no longer an afterthought tagged onto the end of a project by overworked security and operations staff. To maintain a secure estate we must build security best practices in from the start of a project and in a consistent manner.
+
+By using our various build tools it's easier to keep services up to date with the latest security patches and identify security vulnerabilities.
+
+#### Add observability into services
+
+No service runs in isolation and gaining a holistic view of the whole estate presents difficult challenges. There are many open source technologies to help with observability but these need coordinating between services to be successful.
+
+By using our application templates we standardise the capturing of telemetry and tracing. This allow us to provide a common toolset for observing our suite of microservices.
+
+#### Take advantage of mature development patterns
+
+Many problems facing developers in HMPPS have been "solved" and using these patterns gives teams a head start.
+
+By using our various build plugins and libraries we have captured some of these patterns to distribute amongst teams. This allows developers to focus on providing value to end users.
+
+## Application Platform Shared Components
+
+### Project Bootstrapper
+
+The project bootstrapper is a utility for generating new projects and/or configuring build pipelines for projects. It works by defining a project in the `projects.json` file and running a script to create and configure a GitHub project, a CircleCI project, a quay.io project and a Kubernetes namespace.
+
+The bootstrapper can be used to copy an existing Kotlin or Typescript template project or can run against an existing project.
+
+As well as removing the effort required to create a build pipeline it allows us to standardise things like GitHub project configuration.
+
+See the project readme on GitHub for usage details.
+
+#### External links (requires login to GitHub)
+
+GitHub project: https://github.com/ministryofjustice/dps-project-bootstrap
+projects.json file: https://github.com/ministryofjustice/dps-project-bootstrap/blob/main/projects.json
```

Comment:
As the bootstrap GitHub project is not publicly visible we are unable to use embedded links as these are rejected by the link checker (they receive 404s).  I don't like the above at all but not sure of a better way to include these links.